CONTRIBUTING.md

```diff
@@ -148,14 +148,30 @@ During a peer review, reviewers will add a tag to the comment to help the author
 [Kudos]|The reviewer liked this, specifically. Go you.
 [Suggestion]| An optional, alternative approach.
 
-
-
 Adding multiple tags to a comment is fine.  Do not combine tags.
 
 Good: ``[Suggestion][Kudos] This is great, but...``
 
 Bad:  ``[Suggestion|Kudos] This is great, but...``
 
+
+## Pull Request Labels
+Labels may be applied to your pull requests, for the purposes of organization and / or overall 
+status.
+
+|       Label       |                         Explanation                          |
+| ----------------- | ------------------------------------------------------------ |
+Awaiting changes    | On hold until requested changes have been applied
+Bug Fix             | Fixes a bug
+CI change           | Modifies the configuration of Continuous Integration (CI)
+Dependency change   | Changes the project's dependencies
+Documentation       | Modifies project documentation
+Feature             | Adds a new feature
+Refactor            | Refactors an existing system
+Linting             | Adjusts the linting rules
+Policy change       | Modifies contributing or development policies
+Team decision       | Requires the consensus of all core contributors
+
```

Comment:
[Suggestion] Maybe add an "Awaiting review"/"Awaiting CO review" for when the only thing blocking at this point in time is a review. Several PRs have been stalled in the past because a review that blocked hasn't been looked by the reviewing party.

Reply:
i do believe the PRs you are referring to were not clearly communicated as being ready for review. More accurately, PRs  have sat without response to previous reviews / comments which is why they have stalled. 

The addition of "awaiting review" is superfluous, as Github already has a mechanism for requesting reviews. 
![image](https://user-images.githubusercontent.com/3110986/54935581-d2a0c000-4edd-11e9-86f0-9860747d3540.png)